cmd/loadTests.go

```diff
@@ -50,31 +56,44 @@ var (
 	UserCreationTimeMaxPerThread         []time.Duration
 	ResourceCreationTimeMaxPerThread     []time.Duration
 	PipelineRunSucceededTimeMaxPerThread []time.Duration
-	DeploymentSucceededTimeMaxPerThread  []time.Duration
+
+	DeploymentSucceededTimeMaxPerThread     []time.Duration
+	ItsPipelineRunSucceededTimeMaxPerThread []time.Duration
+
 	UserCreationTimeSumPerThread         []time.Duration
 	ResourceCreationTimeSumPerThread     []time.Duration
 	PipelineRunSucceededTimeSumPerThread []time.Duration
 	PipelineRunFailedTimeSumPerThread    []time.Duration
-	DeploymentSucceededTimeSumPerThread  []time.Duration
-	DeploymentFailedTimeSumPerThread     []time.Duration
+
+	DeploymentSucceededTimeSumPerThread     []time.Duration
+	DeploymentFailedTimeSumPerThread        []time.Duration
+	ItsPipelineRunSucceededTimeSumPerThread []time.Duration
+	ItsPipelineRunFailedTimeSumPerThread    []time.Duration
+
 	SuccessfulUserCreationsPerThread     []int64
 	SuccessfulResourceCreationsPerThread []int64
 	SuccessfulPipelineRunsPerThread      []int64
 	SuccessfulDeploymentsPerThread       []int64
-	FailedUserCreationsPerThread         []int64
-	FailedResourceCreationsPerThread     []int64
-	FailedPipelineRunsPerThread          []int64
-	FailedDeploymentsPerThread           []int64
-	frameworkMap                         *sync.Map
-	userComponentMap                     *sync.Map
-	errorCountMap                        map[int]ErrorCount
-	errorMutex                           = &sync.Mutex{}
-	usersBarMutex                        = &sync.Mutex{}
-	resourcesBarMutex                    = &sync.Mutex{}
-	pipelinesBarMutex                    = &sync.Mutex{}
-	deploymentsBarMutex                  = &sync.Mutex{}
-	threadsWG                            *sync.WaitGroup
-	logData                              LogData
+	SuccessfulItsPipelineRunsPerThread   []int64
+
+	FailedUserCreationsPerThread     []int64
+	FailedResourceCreationsPerThread []int64
+	FailedPipelineRunsPerThread      []int64
+	FailedDeploymentsPerThread       []int64
+	FailedItsPipelineRunsPerThread   []int64
+	frameworkMap                     *sync.Map
+	userComponentMap                 *sync.Map
+	userTestScenarioMap              *sync.Map
+	userComponentPipelineRunMap      *sync.Map
+	errorCountMap                    map[int]ErrorCount
+	errorMutex                       = &sync.Mutex{}
+	usersBarMutex                    = &sync.Mutex{}
+	resourcesBarMutex                = &sync.Mutex{}
+	pipelinesBarMutex                = &sync.Mutex{}
+	deploymentsBarMutex              = &sync.Mutex{}
+	itsPipelinesBarMutex             = &sync.Mutex{}
+	threadsWG                        *sync.WaitGroup
+	logData                          LogData
```

Comment:
Hello @naftalysh . I would prefer next time to avoid refactoring like this. If needed, do it in different PR. It is hard to simply see what was actually changed.

Reply:
If you relate to deploymentsBarMutex and itsPipelinesBarMutex, they actually follow the already created Mutexes for the previous test cycles in the user journey - usersBarMutex, resourcesBarMutex and pipelinesBarMutex 
and they relate to the new test cycles for the user journey